nextflow_schema.json

```diff
@@ -564,16 +579,8 @@
         "include_proteins_from_vcf": {
             "type": "string"
         },
-        "config_profile_name": {
-            "type": "string",
-            "default": "Full test profile"
-        },
-        "igenomes_base": {
-            "type": "string",
-            "default": "s3://ngi-igenomes/igenomes/"
-        },
         "multiqc_config": {
```

Comment:
These parameters are currently ungrouped and look like they haven't really been customised at all (no help text etc). At least some are boilerplate so you can copy from the template, eg. this [`multiqc_config`](https://github.com/nf-core/tools/blob/d0a6649cbe7bff98b89a9990955a3fe256e02d55/nf_core/pipeline-template/nextflow_schema.json#L144-L149).